### DISCUSSION TITLE
Add weight to issues and have an ability to sort by Priority and Weight

### DISCUSSION THREAD

**Discussion by oleg-andreyev** [community/community#76417] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Add weight (dran'n'drop) to issues and have an ability to sort by Priority and Weight.

Case:

I have N tickets, with High priority and inside that group I'd like to prioritize issues by weight

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by theoriginalstove** [comment]
+1 as this would be very helpful in using Github Issues and Projects to track/plan while bringing them to some form of parity with every other tool on the market. Pretty sure these have been requested since at least 2015 according to this: https://github.com/isaacs/github/issues/472

**Comment by coach-arne** [comment]
+1 really necessary if you want to use github issues for planning
